README.md

```diff
@@ -1,13 +1,16 @@
 # Demo of Components on this Branch
+<<<<<<< HEAD
```

Comment:
A merge conflict seems to have made its way in. Accept one of the changes and remove the other one.